### PR TITLE
init ChIPseq

### PR DESCRIPTION
**Pull request by mjjeon**

### CODE REVIEW

File: appyters/ChIP_seq/appyter.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "image": "screenshot.png",
+  "authors": [
+    {
+      "name": "Minji Jeon",
+      "email": "minji.jeon1@gmail.com"
+    }
+  ],
+  "url": "https://github.com/MaayanLab/appyter-catalog",
+  "tags": [
+    "ChIP-seq", "RNAseq"
```
Comment (u8sand) [inline appyters/ChIP_seq/appyter.json:16]: In the future, please review the tags already on the catalog. I'll be changing this to `RNA-seq`

Comment (mjjeon) [reply appyters/ChIP_seq/appyter.json:16]: Got it!

**Review by u8sand — CHANGES_REQUESTED** [review]

File: appyters/ChIP_seq/appyter.json (hunk 1 above)
Comment (mjjeon) [reply appyters/ChIP_seq/appyter.json:16]: Can I keep ChIP-seq?

Comment (u8sand) [reply appyters/ChIP_seq/appyter.json:16]: Yeah ChIP-seq is fine

File: appyters/ChIP_seq/igv.css.map [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
[the remark this answers is not in the document]
Comment (mjjeon) [reply appyters/ChIP_seq/igv.css.map:1]: Yeah it is necessary. It is for rendering the genome browser..

File: appyters/ChIP_seq/templates/base.j2 [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% extends "biojupies/base.j2" %}
 {% block head %}
 <title>ChIP-seq analysis pipeline</title>  
-<link rel="shortcut icon" href="https://image.flaticon.com/icons/svg/65/65987.svg"/>
-<link rel="icon" href="https://image.flaticon.com/icons/svg/65/65987.svg">  
+<link rel="shortcut icon" href="../static/analysis.png"/>
+<link rel="icon" href="../static/analysis.png">
```
Comment (u8sand) [inline appyters/ChIP_seq/templates/base.j2:5]: @mjjeon I'll change these for you but please use `{{ url_for('static', 'analysis.png') }}` instead in the future; this mechanism will ensure that these relative paths function regardless of the deployment strategy (for instance, if they are **not submounted onto a fixed prefix**

File: appyters/ChIP_seq/requirements.txt [hunk 4]
```diff
@@ -18,3 +16,5 @@ itables
 igv
 igv-jupyter
 seaborn
+pytest-shutil
```
Comment (u8sand) [inline appyters/ChIP_seq/requirements.txt:19]: Shutil is in the standard library

File: appyters/ChIP_seq/ChIP_seq_analysis.ipynb [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import IFrame\n",
+    "shutil.copyfile(bed_filename, \"./peaks.bed\")        \n",
+    "directory = os.getcwd()\n",
+    "pid = directory.split(\"/\")[-1]\n",
```
Comment (u8sand) [inline appyters/ChIP_seq/ChIP_seq_analysis.ipynb:320]: I don't think this is going to work in production

**Comment by u8sand** [comment]
There are some outstanding issues but i'll address them later today after I've deployed the newer version of the appyter library.

**Pull request merged**